frappe/core/doctype/user_permission/test_user_permission.py

```diff
@@ -11,40 +11,55 @@ class TestUserPermission(unittest.TestCase):
 	def test_apply_to_all(self):
 		''' Create User permission for User having access to all applicable Doctypes'''
 		user = get_user()
-		param = get_params(user, apply = 1)
-		created = add_user_permissions(param)
-		self.assertEquals(created, 1)
+		param = get_params(user, apply_to_all = 1)
+		is_created = add_user_permissions(param)
+		self.assertEquals(is_created, 1)
+
+	def test_for_apply_to_all_on_update_from_apply_all(self):
+		user = get_user()
+		param = get_params(user, apply_to_all=1)
+
+		add_user_permissions(param)
+
```

Comment:
@Anurag810 Add an assert statement here to verify that the user permission doc is created first time.

Reply:
done